core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -328,9 +339,7 @@ protected void drawLaser(Tile tile, Tile target){
         x2 += t2.x;
         y2 += t2.y;
 
-        float fract = 1f - tile.entity.power.graph.getSatisfaction();
-
-        Draw.color(Color.white, Pal.powerLight, fract * 0.86f + Mathf.absin(3f, 0.1f));
```

Comment:
Displaying power status with lines could still be helpful. Is it a problem?

Reply:
no, would actually make sence to keep it in 🤔